docs/Contributing/Building-Fleet.md

```diff
@@ -175,4 +175,35 @@ dlv debug --build-flags '-tags=full' ./cmd/fleet -- serve --dev
 
 It is important to pass the `-tags=full` build flag, otherwise the server will not have access to the asset files.
 
-<meta name="pageOrderInSection" value="100">
\ No newline at end of file
+### Attaching a debugger to a running server
+
+You can also run delve in headless mode, which allows you to attach your preferred debugger client and reuse the same session without having to restart the server:
+
+```
+dlv debug --build-flags '-tags=full' --headless \
+  --api-version=2 --accept-multiclient --continue \
+  --listen=127.0.0.1:61179 ./cmd/fleet -- serve --dev
```

Comment:
this port was chosen somewhat randomly, happy to change it or make it clear that you can change it locally.